InferenceSystem/src/github_utils/github_utils.py

```diff
@@ -0,0 +1,20 @@
+import requests
+import json
+import os
+
+token = os.getenv('GITHUB_ACCESS_TOKEN')
+
+"""Creates pull request within orcasound/aifororcas-livesystem.
+
+parameters:
+title -- The title of pull request
+head -- The name of the branch where your changes are implemented
+base -- The name of the branch you want the changes pulled into
+body -- The description or contents of the pull request
+"""
+def create_pull_request(title, head, base, body):
```

Comment:
```suggestion
def create_pull_request(title, head, base, body):
    """Creates pull request within orcasound/aifororcas-livesystem.

    Args:
        `title`: The title of pull request
        `head`: The name of the branch where your changes are implemented
        `base`: The name of the branch you want the changes pulled into
        `body`: The description or contents of the pull request
    """
```
In Python docstrings look a bit different, they should be inside the function. Also added some formatting so it looks like this:
![image](https://user-images.githubusercontent.com/12860284/137355078-924f83e0-fd88-4555-8d13-99a14b5aa8d5.png)